### DISCUSSION TITLE
Not able to get access to api/frontend

### DISCUSSION THREAD

**Discussion by suprith-hub** [Unleash/unleash#6074] [General]
Error is:{"message":"This endpoint requires an API token. Please add an authorization header to your request with a valid token"} 

I have entered FRONTEND token I created and javascript SDK,. why is this error

**Comment by sighphyre** [comment]
Hey @Era-cell,

It's quite hard to debug this without more information, a full stack trace would be helpful here. When you say the JavaScript SDK, do you mean [this](https://github.com/Unleash/unleash-client-node) or [this](https://github.com/Unleash/unleash-proxy-client-js)?

If you're using the first, then you need a [Client Token](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens), not a Frontend Token.

It may also be a lot easier to get help on these issues through our [community Slack](unleash-community.slack.com) rather than discussions here, the turn around is a lot faster

**Reply by suprith-hub** [reply]
Hey frontend is good, 
but in server side sdk im very confused: I have used python SDK
my code: 
client = UnleashClient(
    url="http://localhost:4242/api",
    app_name='V11',
    custom_headers={'Authorization': '*:development.76998a8514449e1b9b83b8ed631c7a9620489f189db0c54b05ff47c3'})

client.initialize_client()

Unleash Client feature fetch failed due to unexpected HTTP status code.
Unleash Client feature fetch failed due to exception: Unleash Client feature fetch failed!
Traceback (most recent call last):
  File "C:\Users\supri\AppData\Local\Programs\Python\Python311\Lib\site-packages\UnleashClient\api\features.py", line 74, in get_feature_toggles
    raise Exception(
Exception: Unleash Client feature fetch failed!


I am very confused, what is this app-name??? and what is the url ? url of backend server or unleash server and why nothing works

**Reply by sighphyre** [reply]
Hey @Era-cell,

I'm actually quite confused now as well. You mentioned the JavaScript SDK and now Python. Are these different issues? If so, please open a new issue for this. In fact, please take these questions to the Slack channels, it's tremendously difficult to answer these questions without more information. Everything you need should be covered in our docs for [Python](https://gh.getunleash.io/unleash-client-python/unleashclient).

This looks like your token is incorrect, this should be a client token, not a frontend token. But it could be a few things. I would also check that Unleash is running at that URL with cURL, but if you're running it locally that looks correct to me.  

I'm going to close this for now, since this a really difficult place for these questions. Please take this to the Slack or consult the documentation
